### ISSUE TITLE
Run on custom data

### ISSUE THREAD

**Issue by bolin-chen** [nianticlabs/mvsanywhere#5]
Hello, I want to test mvsanywhere on my data. I have a couple of images and the corresponding extrinsics and intrinsics. How can I use these data to the model? Could you provide a minimal available demo? The run_demo.py is too complex and I cannot find how to pass the extrinsics and intrinsics.

**Comment by Golbstein** [comment]
> Hey [@bolin-chen](https://github.com/bolin-chen)
> 
> Great point! Thanks for pointing out this!
> 
> I would like to prepare a small demo for custom images, but I haven't done so yet.
> 
> Meanwhile you can start adapting this code snippet:
> 
> import torch
> 
> import mvsanywhere.options as options
> from mvsanywhere.utils.model_utils import get_model_class, load_model_inference
> 
> # Create and load options
> option_handler = options.OptionsHandler()
> option_handler.parse_and_merge_options()
> opts = option_handler.options
> 
> # Initialize the model
> model_class_to_use = get_model_class(opts)
> model = load_model_inference(opts, model_class_to_use)
> model = model.eval()
> 
> # Reference image data
> cur_data = {
>     "image_b3hw": torch.rand(1, 3, 480, 640).float().cuda(),
>     "cam_T_world_b44": torch.eye(4).unsqueeze(0).float().cuda(),
>     "world_T_cam_b44": torch.eye(4).unsqueeze(0).float().cuda(),
>     "K_matching_b44": torch.rand(1, 4, 4).float().cuda(),
>     "invK_matching_b44": torch.linalg.inv(torch.rand(1, 4, 4)).float().cuda(),
> }
> 
> # Assuming the source data has 7 images
> src_data = {
>     "image_b3hw": torch.rand(1, 7, 3, 480, 640).float().cuda(),
>     "cam_T_world_b44": torch.eye(4).unsqueeze(0).repeat(1, 7, 1, 1).float().cuda(),
>     "world_T_cam_b44": torch.eye(4).unsqueeze(0).repeat(1, 7, 1, 1).float().cuda(),
>     "K_matching_b44": torch.rand(1, 7, 4, 4).float().cuda(),
>     "invK_matching_b44": torch.linalg.inv(torch.rand(1, 7, 4, 4)).float().cuda(),
> 
> 
> outputs = model(
>     phase="test",
>     cur_data=cur_data,
>     src_data=src_data,
>     return_mask=True,
>     num_refinement_steps=1,
> )
> You should change the `cur_data` and `src_data` with your own current and reference data. Then you can call this script with:
> 
> ```
> python3 script_name.py  --config_file configs/models/mvsanywhere_model.yaml  --load_weights_from_checkpoint weights/mvsanywhere_hero.ckpt
> ```
> 
> Hope this helps!

Hey
I think worth mentioning that intrinsic matrix K should represent downsized image by x4 (dividing by 4 the first 2 rows)

Regarding the src images - should they be just the nearest camera position frames or I am missing something?
Thx

**Comment by ttaosci** [comment]
@serizba Thanks for the nice work and easy setup of the environment! One suggestion: It'd be great to remind people that we need to perform ImageNet normalization before feeding the data into the network.
